src/frontend/Komponenter/Personoversikt/RevurderingModal.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from 'react';
+import { Behandlingstype } from '../../App/typer/behandlingstype';
+import {
+    Behandlingsårsak,
+    behandlingsårsaker,
+    behandlingsårsakTilTekst,
+} from '../../App/typer/Behandlingsårsak';
+import { Flatknapp, Hovedknapp } from 'nav-frontend-knapper';
+import { AlertStripeFeil } from 'nav-frontend-alertstriper';
+import UIModalWrapper from '../../Felles/Modal/UIModalWrapper';
+import styled from 'styled-components';
+import { Select } from 'nav-frontend-skjema';
+import { FamilieDatovelger } from '@navikt/familie-form-elements';
+import { Ressurs, RessursStatus } from '../../App/typer/ressurs';
+import { useApp } from '../../App/context/AppContext';
+
+const StyledSelect = styled(Select)`
+    margin-top: 2rem;
+`;
+
+const KnappeWrapper = styled.div`
+    margin-top: 16rem;
+    margin-bottom: 1rem;
+`;
+
+const StyledFamilieDatovelgder = styled(FamilieDatovelger)`
+    margin-top: 2rem;
+`;
+
+const StyledHovedknapp = styled(Hovedknapp)`
+    margin-right: 1rem;
+`;
+
+interface IProps {
+    visModal: boolean;
+    settVisModal: (bool: boolean) => void;
+    fagsakId: string;
+    hentFagsak: () => void;
+    settSkalNavigereTilBehandlingen: (bool: boolean) => void;
+    settKanStarteRevurdering: (bool: boolean) => void;
+}
+
+const RevurderingsModal: React.FunctionComponent<IProps> = ({
+    visModal,
+    settVisModal,
+    fagsakId,
+    hentFagsak,
+    settSkalNavigereTilBehandlingen,
+    settKanStarteRevurdering,
+}) => {
+    const [feilmeldingModal, settFeilmeldingModal] = useState<string>();
+    const [valgtBehandlingstype, settValgtBehandlingstype] = useState<Behandlingstype>(
+        Behandlingstype.REVURDERING
+    );
+    const [valgtBehandlingsårsak, settValgtBehandlingsårsak] = useState<Behandlingsårsak>();
+    const [valgtDato, settValgtDato] = useState<string>();
+
+    const validerKanStarteRevurdering = (): boolean => {
+        return !!(valgtBehandlingstype && valgtBehandlingsårsak && valgtDato);
+    };
+    const { axiosRequest } = useApp();
+
+    const opprettRevurdering = (fagsakId: string) => {
+        settFeilmeldingModal('');
+        if (validerKanStarteRevurdering()) {
+            settKanStarteRevurdering(false);
+            settVisModal(false);
+            axiosRequest<Ressurs<void>, { fagsakId: string }>({
```

Comment:
Småpirk, men burde kanskje ha med mer enn `fagsakId`her? Evt definere en egen type lik dto'en backend forventer
```suggestion
            axiosRequest<Ressurs<void>, { fagsakId: string,  behandlingsårsak: ??, kravMottatt: ?? }>({
```